mne/commands/mne_anonymize.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""Anonymize .fif file.
+
+You can do for example:
+
+$ mne anonymize sample_audvis_raw.fif
+"""
+
+# Authors : Dominik Krzeminski
+
+import sys
+import mne
+
+
+def mne_anonymize(fif_fname):
+    """ Call *anonymize_info* on fif file and save.
+    Parameters
+    ----------
+    fif_fname : str
+        Raw fif File
+    """
+    raw = mne.io.read_raw_fif(fif_fname, preload=True)
+    mne.io.anonymize_info(raw.info)
+    raw.save(fif_fname, overwrite=True)
+
+
+def run():
+    """Run *mne_anonymize* command."""
+    from mne.commands.utils import get_optparser
+
+    parser = get_optparser(__file__)
+
+    options, args = parser.parse_args()
+    if len(args) != 1:
+        parser.print_help()
+        sys.exit(1)
+
+    fname = args[0]
```

Comment:
please use the --file option.

Reply:
@agramfort actually here the use of argument was intended, as actually file name is something what we require from user to pass. And from `optparse` documentation:
"note that the phrase “required option” is self-contradictory in English." https://docs.python.org/2/library/optparse.html which kind of makes sense imho